acme_diags/driver/qbo_driver.py

```diff
@@ -194,6 +194,7 @@ def run_diag(parameter):
                     test_json[key] = list(test[key])
                     ref_json[key] = list(ref[key])
 
+        parameter.output_file = 'qbo_diags'
```

Comment:
Other diags have variables put into the output file name, but QBO diags seems fairly independent of variables (e.g., variable is always `U`,`V`, etc.) . Are there any we should include? Compare to ENSO diags: https://github.com/E3SM-Project/e3sm_diags/blob/master/acme_diags/driver/enso_diags_driver.py#L270

Reply:
U is the only variable we will apply for QBO. In my test, the variable name is not shown up in the viewer though: https://acme-viewer.llnl.gov/zhang40/qbo_diags_acme1_m2m/viewer/qbo/index.html.

Reply:
I haven't tested with the new change though.

Reply:
@chengzhuzhang Oh that's interesting. I see `ERA-Interim` under `Variable` when I use my `run` script. 

Should I add `U` to the `output_file` name?

Reply:
It seems like case-id is shown in your viewer instead of variable? I was running a model vs model case.

Reply:
Well, ENSO uses `case_id` too: https://github.com/E3SM-Project/e3sm_diags/blob/master/acme_diags/driver/default_diags/enso_diags_model_vs_model.cfg#L3, https://github.com/E3SM-Project/e3sm_diags/blob/master/acme_diags/viewer/enso_diags_viewer.py#L59

Reply:
I found another problem with plotting. And will try tweak the viewer option some. I will update in another PR.